examples/zkquiz/README.md

```diff
@@ -52,3 +52,34 @@ This will:
 3. Pay & submit proof to aligned for verification
 4. Wait for proof to be verified in aligned
 5. Claim NFT if proof is verified
+
+## Testing locally
+
+If you want to test the zk quiz on a local network follow these steps:
+
+1. Setup Aligned locally following [this guide](../../docs/3_guides/6_setup_aligned.md)
+
+3. Move into the zkquiz example:
+    ```
+    cd examples/zkquiz
+    ```
+
+4. Deploy the ZKQuiz verifier contract, and locate the `CONTRACT_ADDRESS` from its output:
+    ```
+    make deploy_verifier_devnet
+    ```
+
+
+5. Run the quiz:
+    ```
+    CONTRACT_ADDRESS=<VERIFIER_CONTRACT_ADDRESS> make answer_quiz_local
+    ```
+
+> [!NOTE]
+> Make sure you send another proof alongside the ZKQuiz proof, as the batcher will not create a batch with only 1 proof.
+> 
+> For example you could do:
+> ```
+> make batcher_send_risc0_burst
+> ```
```

Comment:
fortunately the PR that fixed the functionality of merkle trees with only one leaf was merged yesterday, so this note shouldn't be necessary. Make sure to test it works correctly without the need to send more proofs

Reply:
removed comment